src/apirequest.ts

```diff
@@ -160,6 +160,9 @@ async function createAPIRequestAsync<T>(parameters: APIRequestParams) {
   // query
   parameters.pathParams.forEach(param => {
     delete params[param];
+    if (parameters.context._options.params) {
```

Comment:
Is it worth being defensive and checking:

```
if (parameters.context && parameters.context._options && parameters.context._options.params) {}
```

Reply:
Good call!